rc/filetype/go.kak

```diff
@@ -134,7 +134,7 @@ define-command -hidden go-insert-on-new-line %[
             try %[ execute-keys -draft k<a-x>s^\h+<ret>"xy ] catch %[ reg x '' ]
             try %[
                 # Validate previous line and that it is not closed yet.
-                execute-keys -draft k<a-x> <a-k>^<c-r>x.*\{\h*\(?\h*$<ret> J}iJ<a-x> <a-K>^<c-r>x(\)?\h*\})$<ret>
+                execute-keys -draft k<a-x> <a-k>^<c-r>x.*\{\h*\(?\h*$<ret> j}iJ<a-x> <a-K>^<c-r>x(\)?\h*\}.*)$<ret>
```

Comment:
Instead of `.*` I'd normally use `[^\n]`. Actually, we know that the selection is a line, so why not just omit the `.*$`, since it will always match. Also I don't think the capturing parentheses do anything here.

```suggestion
                execute-keys -draft k<a-x> <a-k>^<c-r>x.*\{\h*\(?\h*$<ret> j}iJ<a-x> <a-K>^<c-r>x\)?\h*\}<ret>
```